scripts/mint.ts

```diff
@@ -20,6 +20,29 @@ const client = await SigningCosmWasmClient.connectWithSigner(
   wallet
 );
 
+async function mint() {
```

Comment:
Maybe renaming this something like `mint_whitelist()` or `test_whitelist()` would be good to prevent it from being used instead of `mint --to`.